src/function/table.rs

```diff
@@ -153,23 +217,27 @@ impl Table {
             self.n_stale += 1;
             let k = mem::take(&mut inp.data);
             let new_offset = self.vals.len();
-            self.vals.push((
+            self.vals.push(Row::new(
                 Input::new(k),
                 TupleOutput {
                     value: next,
                     timestamp: ts,
                 },
+                unextractable,
```

Comment:
If I understand the behavior here correctly, it would throw away the current `unextractable` and `subsumed` status of the row and always use the passed-in statuses instead. This does not feel right to me. For example, if a row is originally marked as unextractable but not subsumed, and later someone calls insert_and_merge with `unextractable=false` but `subsumed=true`. This would make the row extractable again. Instead, the correct behavior is to view unextractable and subsumed as a boolean lattice and take the lattice join of the old and current status (i.e., `||`).